ai_genomics/utils/save_plotting.py

```diff
@@ -0,0 +1,88 @@
+"""
+utils.save_plotting
+Utils for easier exporting of altair charts
+"""
+from altair_saver import save
+from selenium import webdriver
+from webdriver_manager.chrome import ChromeDriverManager
+import os
+from typing import Iterator
+from pathlib import Path
+
+FIGURE_PATH = Path("outputs/")
```

Comment:
This saves the files to a new `outputs` folder in the directory where we are running the script. What about...

```
from ai_genomics import PROJECT_DIR

etc.

FIGURE_PATH = Path(f"{PROJECT_DIR}/outputs/figures")
```